browserassets/js/browserOutput.js

```diff
@@ -913,25 +943,20 @@ $(function() {
         } else {
             xmlHttp = new ActiveXObject("Microsoft.XMLHTTP");
         }
-        xmlHttp.open('GET', 'browserOutput.css', false);
+        xmlHttp.open('GET', 'http://cdn.goonhub.com/css/browserOutput.css', false);
```

Comment:
It is intentional that this has the cdn url here. Using the relative url keeps freezing your client indefinitely, at least locally.